massa-bootstrap/src/listener.rs

```diff
@@ -43,6 +43,7 @@ impl BootstrapTcpListener {
         if addr.is_ipv6() {
             socket.set_only_v6(false)?;
         }
+        socket.set_nonblocking(true)?;
```

Comment:
@Ben-PH Can this introduce unexpected behaviours?
Thinking of something like failing with a WouldBlock would be raised as an error somewhere where we would not want it to.

Reply:
This is needed for the mio-polling system, which depends on the socket being non-blocking.

Reply:
...my understanding might be outdated, so mught be worth trying to prove me wrong here: See if we need this line at all.